shopelectro/tests/tests_views.py

```diff
@@ -169,6 +186,71 @@ def test_product_tag_linking(self):
         for link in property_links:
             self.assertContains(response, link)
 
+    def test_pagination_numbering(self):
+        page_number = 1
+        response = self.get_category_page(query_string={'page': page_number})
+        self.assertEqual(response.context['paginated_page'].number, page_number)
+
+    def test_pagination_products_number(self):
+        """
+        @todo #302:30m Implement test case for pagination logic.
+         Products number changes in depend on page number.
```

Comment:
`Products count depends on page number.`

"count" is more good, then "number"

Reply:
In this case "number"/"quantity" are better, so if "number" confusing you, then let's use here "quantity"

Reply:
@artemiy312 , added it to discuss list. Leave `number` here